### PR TITLE
Add approvers to the forthcoming ZTP profile area

### PR DESCRIPTION
**Pull request by lack**
This defines a ZTP approver alias as well as granting approver access to
feature-configs/ZTP for that alias.

### CODE REVIEW

**Comment by yuvalk** [comment]
so, this one will need the same hack we did with the previous, similar, pr, 
due to verify-owners plugin bug that doesnt support adding alias and using it in the same PR
https://github.com/kubernetes/test-infra/issues/21509

**Comment by yuvalk** [comment]
is ZTP really a feature-config ?
I *think* it should be on it's own directory
but not sure what content will exactly go there?

**Comment by lack** [comment]
> so, this one will need the same hack we did with the previous, similar, pr,
> due to verify-owners plugin bug that doesnt support adding alias and using it in the same PR
> [kubernetes/test-infra#21509](https://github.com/kubernetes/test-infra/issues/21509)

It sure will.  What was the hack?

**Comment by lack** [comment]
> is ZTP really a feature-config ?
> I _think_ it should be on it's own directory
> but not sure what content will exactly go there?

After some conversation with @ijolliffe I got the path wrong.  Moved from `/feature-config/ZTP` to `/ztp`

**Comment by yuvalk** [comment]
> > is ZTP really a feature-config ?
> > I _think_ it should be on it's own directory
> > but not sure what content will exactly go there?
> 
> After some conversation with @ijolliffe I got the path wrong. Moved from `/feature-config/ZTP` to `/ztp`
so just to make sure,
this is going to be used as place to consolidate the existing ansible playbooks,
and in the future as reference for the ACM-AI-Metal3 ZTP approach?


> > so, this one will need the same hack we did with the previous, similar, pr,
> > due to verify-owners plugin bug that doesnt support adding alias and using it in the same PR
> > [kubernetes/test-infra#21509](https://github.com/kubernetes/test-infra/issues/21509)
> 
> It sure will. What was the hack?

triple check the PR is fine and wont block CI in the future
then ask an owner to remove the `do-not-merge/invalid-owners-file` label manually
then PR will merge (once it have all the labels etc)

**Review by MarSik — APPROVED** [review]

**Comment by yuvalk** [comment]
@MarSik please also manually remove the do-not-merge label, so this can merge (see my comment above)

**Review by imiller0 — APPROVED** [review]
/approve

**Comment by yuvalk** [comment]
> /remove do-not-merge/invalid-owners-file

no, only an org owner can do that (@MarSik , @cynepco3hahue , I've used the generous help of @sreichar with the previous PR)

and I think this PR is fine for that manual override too (the risk btw: is blocking all other PRs if something is bad here, but I've manually validated the names are all org members)

**Comment by fromanirh** [comment]
/remove-invalid-owners-file

> > /remove do-not-merge/invalid-owners-file
> 
> no, only an org owner can do that (@MarSik , @cynepco3hahue , I've used the generous help of @sreichar with the previous PR)
> 
> and I think this PR is fine for that manual override too (the risk btw: is blocking all other PRs if something is bad here, but I've manually validated the names are all org members)

for the record, I did the same manual validation before adding my approval.

**Comment by yuvalk** [comment]
> /verify-owners
> /remove do-not-merge/invalid-owners-file

again, there is a bug with the verify-owners plugin. it cannot merge a PR that add an ALIAS and try to use it in an OWNER file in the same PR.

you need to manually remove the label. the bot cant do it

**Comment by yuvalk** [comment]
thanks @cynepco3hahue !

**Pull request merged**
